ssl/src/main/scala/org/scalanative/ssl/OpenSSLSessionContext.scala

```diff
@@ -20,21 +20,20 @@ import java.util.Enumeration
 import java.util.HashMap
 import javax.net.ssl.SSLSession
 import javax.net.ssl.SSLSessionContext
-import scala.collection.immutable.ArraySeq
 
 abstract class OpenSSLSessionContext extends SSLSessionContext {
 
-  private[this] val sessions = new HashMap[ArraySeq.ofByte, SSLSession]
+  private[this] val sessions = new HashMap[Array[Byte], SSLSession]
```

Comment:
Thanks, unfortunately this will not work. The hashmap keys should be defined on equality of the byte content, and unfortunately `Array` compares by reference equality.

Reply:
Oh, silly me, of course :facepalm: 
But how about `ByteBuffer`? :crossed_fingers:

Reply:
Nice, I believe you are right! `ByteBuffer` hadn't occurred to me. Thanks!